docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+PostgreSQL:
+  image: sameersbn/postgresql:9.6-2
+  hostname: postgres
+  restart: always
+  ports:
+    - "5432:5432"
+  environment:
+    - DEBUG=false
```

Comment:
Can we have a `*.env` file that contains all the environment variables to be passed in the container? See docker documentation. Quite straightforward to do  that.

Reply:
Nice idea ! I forgot about this. Doing !